mailet/icalendar/pom.xml

```diff
@@ -109,6 +109,13 @@
         <dependency>
             <groupId>org.mnode.ical4j</groupId>
             <artifactId>ical4j</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.cache</groupId>
```

Comment:
no longer use ?

Reply:
The API is required as ICal4J uses an implem of this.

If I remove it, it breaks...

Reply:
from ical4j pom.xml : 
```
    <dependency>
      <groupId>javax.cache</groupId>
      <artifactId>cache-api</artifactId>
      <version>1.0.0</version>
      <scope>compile</scope>
      <optional>true</optional>
    </dependency>
```

if it's really breaking, then it's not optional. Please open a bug on ical4j

Reply:
i created an issuse. see https://github.com/linagora/james-project/pull/1564